index.js

```diff
@@ -126,6 +126,35 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: contextMenuLabel,
+  context: [
+    cm.URLContext(['*.google.com']),
+    //TODO?: create a variable with all supported hrefs and reference it in this SelectorContext
```

Comment:
Something like `const supportedGoogleSelectors = '[href*="youtube.com%2F"], [href*="youtu.be%2F"], [href*="vine.co%2F"], [href*="vimeo.com%2F"]'` at the top?